neurom/features/neuritefunc.py

```diff
@@ -308,14 +326,50 @@ def partition_asymmetries(neurites, neurite_type=None):
 
 def partition_pairs(neurites, neurite_type=None):
     '''Partition pairs at bifurcation points of a collection of neurites.
-    Partition pait is defined as the number of bifurcations at the two
+    Partition pair is defined as the number of bifurcations at the two
     daughters of the bifurcating section'''
     return map(bifurcationfunc.partition_pair,
                iter_sections(neurites,
                              iterator_type=Tree.ibifurcation_point,
                              neurite_filter=is_type(neurite_type)))
 
 
+def partition_asymmetry_lengths(neurites, neurite_type=None):
+    '''Variant of partition asymmetry at bifurcation points of a collection of neurites'''
+    tot_lengths = total_length_per_neurite(neurites)
+    output = []
+    for i, neurite in enumerate(iter_neurites(neurites, filt=is_type(neurite_type))):
+        p_asymmetry = map(bifurcationfunc.partition_asymmetry_length,
+                          iter_sections(neurite,
+                                        iterator_type=Tree.ibifurcation_point,
+                                        neurite_filter=is_type(neurite_type)))
+        output += list(np.array(list(p_asymmetry)) / tot_lengths[i])
+    return output
+
+
+def sibling_ratios(neurites, neurite_type=None, method='first'):
+    '''Sibling ratios at bifurcation points of a collection of neurites.
+    The sibling ratio is the ratio between the diameters of the
+    smallest and the largest daughter. It is a real number between
+    0 and 1. Method argument allows one to consider mean diameters
+    along the daughter section instead of diameter of the first point. '''
+
+    return map(lambda bif_point: bifurcationfunc.sibling_ratio(bif_point, method),
+               iter_sections(neurites,
+                             iterator_type=Tree.ibifurcation_point,
+                             neurite_filter=is_type(neurite_type)))
+
+
+def diameter_power_relations(neurites, neurite_type=None, method='first'):
+    '''Calculate the diameter power relation at a bifurcation point
+    as defined in https://www.ncbi.nlm.nih.gov/pubmed/18568015'''
+
+    return map(lambda bif_point: bifurcationfunc.diameter_power_relation(bif_point, method),
+               iter_sections(neurites,
+                             iterator_type=Tree.ibifurcation_point,
+                             neurite_filter=is_type(neurite_type)))
```

Comment:
I think it is clearer without the map.
```suggestion
    return (bifurcationfunc.diameter_power_relaion(bif_point, method)
           for bif_point in iter_sections(neurites,
                             iterator_type=Tree.ibifurcation_point,
                             neurite_filter=is_type(neurite_type))))
```